net-misc/dibbler/files/dibbler-1.0.1-clang-16-musl-fix.patch

```diff
@@ -0,0 +1,63 @@
+From: Brahmajit Das <brahmajit.xyz@gmail.com>
+Date: Wed, 31 May 2023 06:27:08 +0000
+Subject: [PATCH] Fix build with clang 16
+
+Especially on musl libc
+
+Signed-off-by: Brahmajit Das <brahmajit.xyz@gmail.com>
+--- a/Port-linux/dibbler-client.cpp
++++ b/Port-linux/dibbler-client.cpp
+@@ -26,7 +26,7 @@ using namespace std;
+ 
+ #define IF_RECONNECTED_DETECTED -1
+ 
+-extern pthread_mutex_t lock;
++extern pthread_mutex_t lock_t;
```

Comment:
Isn't x_t reserved?

Reply:
Huh, I wasn't sure. I mean I'm renameing it, so I can just rename it something else. How about plock or p_lock?

Reply:
- https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html
- https://devblogs.microsoft.com/oldnewthing/20230109-00/?p=107685 (this doesn't include everything)

I'd do 'plock'.

Reply:
done